pkg/controller/common/helpers.go

```diff
@@ -285,12 +298,41 @@ func ValidateIgnition(ignconfig interface{}) error {
 	}
 }
 
+// InSlice search for an element in slice and return true if found, otherwise return false
+func InSlice(elem string, slice []string) bool {
```

Comment:
can be unexported `inSlice` till we find a use for it (if any)

Reply:
thanks for catching this, this was carried from work done in https://github.com/openshift/machine-config-operator/pull/1850. I missed updating inArray() call in update.go to InSlice(). Fixed https://github.com/openshift/machine-config-operator/pull/1941/files#diff-06961b075f1753956d802ba954d2cfb5R694